src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+import java.util.Random;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class Duke {
+    private static File save;
+
+    public static String addTaskSalutation(String item) {
+        Random rand = new Random();
+        String out = "";
+        switch(rand.nextInt(3)) {
+        case 0:
+            out = "Right away sir. I have added " + item + " to your list of tasks.\n";
+            break;
+        case 1:
+            out = "Understood sir. " + item + " has been added to your list of tasks.\n";
+            break;
+        case 2:
+            out = "Sir, I can confirm that " + item + " has been added to your list of tasks.\n";
+            break;
+        default:
+            break;
+        }
+        return out;
+    }
+    public static void line() {
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    public static void showList(Tasks user) {
+        for (int i = 0; i < user.listLength; i++) {
+            System.out.println("\t" + (i + 1) + ". " + user.list[i] + "\n");
+        }
+    }
+    public static void addTodo(String input, Tasks user) {
+        user.list[user.listLength] = new Todo(input.substring(5));
+        user.listLength ++;
+        user.tasksIncomplete ++;
+    }
+    public static void addDeadline(String input, Tasks user) {
+        user.list[user.listLength] = new Deadline(input.substring(9, input.indexOf("/") - 1), input.substring(input.indexOf("/") + 1), input.substring(9));
+        user.listLength ++;
+        user.tasksIncomplete ++;
```

Comment:
You have repeated uses of `user.list[x]` and other variables that keep track of the array position to fill. This makes the code slightly more complicated than necessary.

You could simplify the code further by doing without the array index (in this case `listLength`).  
Try `ArrayList` so that you could add items to a growing list.